build.csx

```diff
@@ -0,0 +1,225 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+#r "System.Xml.Linq"
+
+using System.Runtime.Serialization.Json;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Linq;
+using static SimpleTargets;
+
+static var solutionName = "FakeItEasy";

```

Comment:
I'm not a fan of const for anything which isn't logically constant, but I don't mind too much either way.